lang/cs/Org.Apache.REEF.Common/Services/ServicesConfigurationOptions.cs

```diff
@@ -27,5 +27,10 @@ public class ServicesConfigurationOptions
         public class Services : Name<string>
         {
         }
+
+        [NamedParameter]
+        public class ServiceConfigString : Name<string>
```

Comment:
If I move WrapServiceConfigAsString into AllocatedEvaluator class, this one has to remain public as it is used cross modules.